assets/js/script.js

```diff
@@ -1,97 +1,101 @@
 $(document).ready(function() {
-    var letter1Choice = "black"
-    var letter2Choice = "black"
+  var el = {
```

Comment:
The purpose of this pattern is so that 

1. a future developer can see at a glance what elements are being manipulated by this code
1. if the html gets out of sync (which it often does), you get a warning in the console that an element your code depends on can no longer be found.